### PR TITLE
Feature/cce 353

### PR DESCRIPTION
**Pull request by saadmhybris**
Update markup for card status

### CODE REVIEW

File: src/less/components/_data.less [hunk 1]
```diff
@@ -63,7 +63,17 @@
 // Markup:
 //<p class="statusDataValue">
 //  <span class="active">
-//    Active
+//    Active Label
+//  </span>
+//</p> <br />
+//<p class="statusDataValue">
```
Comment (taifunhaeberle) [inline src/less/components/_data.less]: Please check the task again. The task is to get rid off the statusDataValues and use the Bootstrap labels ( http://getbootstrap.com/components/#labels )

Comment (jeremytrawls) [inline src/less/components/_data.less]: @jensru  can you please clarify the desired changes?

Comment (jensru) [inline src/less/components/_data.less]: @jeremytrawls people where confused, that we used two versions of displaying color, inside cards. Do you have any concerns with removing "statusDataValue"?

Comment (saadmhybris) [inline src/less/components/_data.less]: So I have replaced the statusDataValue with bootstrap label in the card examples and and also the Status Value Label. Are we all set here?

@jensru @taifunhaeberle

**Pull request merged**